spec/lovemocks.lua

```diff
@@ -0,0 +1,13 @@
+--love mocks
```

Comment:
We don't need this file if we have spec/lovemock.lua, right? Should merge/discard this file.